File: app/src/main/java/com/mobile/model/Alert.java
package com.mobile.model;

public class Alert {

    String id, title, body, urlTitle, url;

    public String getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getBody() {
        return body;
    }

    public String getUrlTitle() {
        return urlTitle;
    }

    public String getUrl() {
        return url;
    }

    public boolean isDismissable() {
        return dismissable;
    }

    boolean dismissable;

    public Alert() {
    }


}

File: app/src/main/java/com/mobile/model/Alert.kt
package com.mobile.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
class Alert(val id: String? = null,
            val title: String? = null,
            val body: String? = null,
            val urlTitle: String? = null,
            val url: String? = null,
            var dismissible: Boolean? = null,
            val dismissButton: Boolean? = false,
            val dismissButtonText: String? = null,
            val dismissButtonWebhook: String? = null

) : Parcelable
